src/main/java/edu/cornell/kfs/module/purap/document/service/impl/PurchaseOrderTransmissionMethodDataRulesServiceImpl.java

```diff
@@ -16,14 +15,16 @@
  */
 public class PurchaseOrderTransmissionMethodDataRulesServiceImpl implements PurchaseOrderTransmissionMethodDataRulesService{
 	
+	private PostalCodeValidationService postalCodeValidationService;

```

Comment:
we might want to make these protected (I know we've done that a lot in base code to facilitate being able to override)